Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -629,4 +629,20 @@ final class AttributeTests: XCTestCase {
       """
     )
   }
+
+  func testAttachedExtensionAttribute() {
+    assertParse(
+      """
+      @attached(extension)
+      macro m()
+      """
+    )
+
+    assertParse(
+      """
+      @attached(extension, names: named(test))
+      macro m()
+      """
+    )
+  }
```

Comment:
`conformances:` as well? Shouldn't really matter since it's just parsed as a custom attribute anyway, but good to have the test case IMO.